modules/code-editor/src/views/full/LiteEditor/index.tsx

```diff
@@ -26,13 +48,14 @@ const argsToInterface = (params?: Parameters[]) => {
   return (params ?? []).filter(Boolean).map(x => ({ name: snakeToCamel(x.name), type: x.type }))
 }
 
-interface Props {
-  onChange: (code: string) => void
-  args?: Parameters[]
-  code: string
-  maximized: boolean
-  displayed: boolean
-  bp: any
+function findLastIndex<T>(array: Array<T>, predicate: (value: T, index: number, obj: T[]) => boolean): number {
```

Comment:
oh, right, this doesn't exist in js.. Just for fun, I think the following would have worked too

``` ts
const revIdx = array.reverse().findIndex(predicate)
return revIdx === -1 ? revIdx : array.length - 1 - revIdx
```

Reply:
Yup, I weighted both approach and made a choice